<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogonDlg</class>
 <widget class="QDialog" name="LogonDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(255, 255, 255, 150);</string>
  </property>
  <widget class="QWidget" name="_pParentWidget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>456</width>
     <height>280</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(148, 218, 255);</string>
   </property>
   <widget class="QPushButton" name="_pCapLoginBtn">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>10</y>
      <width>135</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="_pCapRegisterBtn">
    <property name="geometry">
     <rect>
      <x>227</x>
      <y>10</y>
      <width>135</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="_pLoginWidget" native="true">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>360</y>
     <width>450</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLineEdit" name="_pLoginIdEdit">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>1</y>
      <width>235</width>
      <height>36</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>235</width>
      <height>36</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>235</width>
      <height>36</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="_pLoginPasswordEdit">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>71</y>
      <width>235</width>
      <height>36</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>235</width>
      <height>36</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>235</width>
      <height>36</height>
     </size>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="_pLoginBtn">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>120</y>
      <width>120</width>
      <height>36</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>36</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>36</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="_pLoginIdLab">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>1</y>
      <width>36</width>
      <height>36</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="_pLoginExitBtn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>120</width>
      <height>36</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>36</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>36</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="_pLoginPasswordLab">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>71</y>
      <width>36</width>
      <height>36</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="_pRegisterWidget" native="true">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>40</y>
     <width>450</width>
     <height>283</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="_pRegisterPhoneLab">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>13</y>
      <width>55</width>
      <height>12</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>55</width>
      <height>12</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>55</width>
      <height>12</height>
     </size>
    </property>
    <property name="text">
     <string>手 机 号：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="_pRegisterPhoneEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>1</y>
      <width>270</width>
      <height>32</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>270</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>270</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="_pRegisterPasswordEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>46</y>
      <width>270</width>
      <height>32</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>270</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>270</width>
      <height>32</height>
     </size>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="_pRegisterPasswordLab">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>56</y>
      <width>55</width>
      <height>12</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>55</width>
      <height>12</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>55</width>
      <height>12</height>
     </size>
    </property>
    <property name="text">
     <string>密    码：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="_pRegisterNameEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>93</y>
      <width>270</width>
      <height>32</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>270</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>270</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="_pRegisterNameLab">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>101</y>
      <width>55</width>
      <height>12</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>55</width>
      <height>12</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>55</width>
      <height>12</height>
     </size>
    </property>
    <property name="text">
     <string>姓    名：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="_pRegisterAddressEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>139</y>
      <width>270</width>
      <height>32</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>270</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>270</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="_pRegisterAddressLab">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>148</y>
      <width>55</width>
      <height>12</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>55</width>
      <height>12</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>55</width>
      <height>12</height>
     </size>
    </property>
    <property name="text">
     <string>地    址：</string>
    </property>
   </widget>
   <widget class="QLabel" name="_pRegisterDateLab">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>193</y>
      <width>55</width>
      <height>12</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>55</width>
      <height>12</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>55</width>
      <height>12</height>
     </size>
    </property>
    <property name="text">
     <string>出生日期：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="_pRegisterBtn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>246</y>
      <width>120</width>
      <height>36</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>36</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>36</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="_pRegisterExitBtn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>246</y>
      <width>120</width>
      <height>36</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>36</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>36</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="_pRegisterUserLab">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>195</y>
      <width>49</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>*使用者</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="deBirthday">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>231</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>32</height>
     </size>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QCalendarWidget" name="_pCalendarWidget">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>460</y>
     <width>248</width>
     <height>197</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
